frontend/src/reducer/currentWorkspace.js

```diff
@@ -168,6 +169,37 @@ export default function currentWorkspace (state = defaultWorkspace, action) {
         recentActivityList: state.recentActivityList.filter(c => !action.workspaceContentList.some(cc => c.id === cc.content_id))
       }
 
+    case `${RESTORE}/${WORKSPACE_CONTENT_SHARE_FOLDER}`:
+    case `${ADD}/${WORKSPACE_CONTENT_SHARE_FOLDER}`:
+      return {
+        ...state,
+        recentActivityList: [
+          ...action.workspaceShareFolderContentList.map(c => serializeContent(c)),
```

Comment:
the serialize process has been updated, you need to use the new process

see:
- the new function: https://github.com/tracim/tracim/blob/develop/frontend_lib/src/helper.js#L375
- usage example: https://github.com/tracim/tracim/blob/develop/frontend/src/reducer/user.js#L50

See with gbonaspetti for questions